src/transformers/extractObjectProperties.js

```diff
@@ -1,4 +1,5 @@
 const t = require('@babel/types');
+const generateBooleanArgument = require('./generateBooleanArgument');
```

Comment:
As this is only used here you can keep it in this file, since it's not such a big change it can stay in the for loop

Reply:
Do you mind swapping for look for a `reduce`? Since you are reducing the old arguments into new once it's kinda makes sense. If not that's cool and I will simple remove the file and inline the logic

Reply:
If so, that should be it's own thing, as that isn't needed to fix the issue.

Reply:
I've inlined the logic and this should be it for this PR